src/main/kotlin/blackjack/domain/player/GamePlayers.kt

```diff
@@ -0,0 +1,43 @@
+package blackjack.domain.player
+
+import blackjack.domain.BlackjackGameResult
+import blackjack.domain.DealerResult
+import blackjack.domain.PlayerResult
+import blackjack.domain.card.Cards
+
+class GamePlayers(val value: List<Player>, val dealer: Dealer = Dealer()) {
+    fun receiveCard(getCards: () -> Cards) {
+        value.plus(dealer).forEach {
```

Comment:
value를 player로 변경하면 좋지 않을까하네요~

추가로 players + dealer 로도 변경이 가능해보입니다

```suggestion
        players+dealer.forEach {
```

Reply:
처음에는 말씀주신 것 처럼 `dealer`도 `players`에 함께 두고 `List<GamePlayer>`로 가지고 작업을 했었는데요,
작업 중에 `dealer`나 `players` 를 따로 사용해야하는 일들이 생기더라구요🤔
그래서 결국 따로 두되 필요한 곳에서 합쳐서 사용하는 로직으로 가게되었는데요
지금보니 `players.plus(dealer)` 처럼 함께 필요한 케이스에 대해 별도 함수로 추출해서 사용해도 좋을 것 같네요,

다만 `OutputView` 쪽에서는 `dealer` 뒤에 `players`들이 추가되야하는 니즈가 있는데 도메인 내에 둘수는 없을 것 같아서
해당 니즈가 있는 곳에서는 별도로 조합해서 사용해야 할 것 같습니다.

혹시 좋은 의견이 있다면 주셔도 좋을 것 같아요🙌

Reply:
해당 부분에 대해서는 현재로써 좋은 그림이기 때문에 의견 없습니다 👍👍